### PR TITLE
docs: Update the Test Doc 

### PR DESCRIPTION
**Pull request by shubham-cmyk**
Fix : https://github.com/kyverno/chainsaw/issues/236
Fix : https://github.com/kyverno/chainsaw/issues/238

### CODE REVIEW

File: website/docs/tests/with-test.md [hunk 1]
```diff
@@ -38,6 +41,10 @@ metadata:
 spec:
   # this timeout applies only to the test (and test steps if not overridden)
   timeout: 10s
+  skip: true
```
Comment (eddycharly) [inline website/docs/tests/with-test.md:44]: skip the test ?

File: website/docs/tests/with-test.md [hunk 2]
```diff
@@ -38,6 +41,10 @@ metadata:
...
+  concurrent: false
+  skipDelete: true
```
Comment (eddycharly) [inline website/docs/tests/with-test.md:46]: skip deletion ?

File: website/docs/tests/with-test.md [hunk 3]
```diff
@@ -38,6 +41,10 @@ metadata:
...
+  namespace: custom-namespace
```
Comment (eddycharly) [inline website/docs/tests/with-test.md:47]: why use an explicit namespace ?

**Comment by shubham-cmyk** [comment]
@eddycharly  

PTAL

**Review by eddycharly — APPROVED** [review]

**Pull request merged**